### DISCUSSION TITLE
Building apps with multiple webviews

### DISCUSSION THREAD

**Discussion by kishaningithub** [tauri-apps/tauri#3499] [Q&A] [answered]
What does it take to build something like https://getferdi.com/ in Tauri?

Alternate ways to build that in Tauri without using multiple webviews are also welcome

PS - I am very new to Tauri.

**Comment by nothingismagick** [comment, marked as answer]
This a current topic of research, but the feature you (and others) are requesting is unavailable at the moment - and there is no ETA. When we have updates, we will inform the community and notify within this discussion.

**Reply by kishaningithub** [reply]
Thanks :-) Linking here the issue threads here on the topic for easier reference

1. https://github.com/tauri-apps/tauri/issues/2709
2. https://github.com/tauri-apps/tauri/issues/2975
